awsx/ecs/ec2Service.ts

```diff
@@ -79,7 +79,7 @@ export class EC2Service extends schema.EC2Service {
                     args.loadBalancers ?? taskDefinition?.loadBalancers,
                 waitForSteadyState: !utils.ifUndefined(
                     args.continueBeforeSteadyState,
-                    false,
+                    true,
```

Comment:
Actually, is this correct ... there's a `!` before the ifUndefined

If continue not defined, we use true, so wait is false.
I think this was actually already correct: if continue not defined, use false, so wait is true.

Not very clear either way 🤨

Reply:
fixed!